src/mcmini_private.cpp

```diff
@@ -34,6 +34,26 @@ sem_t mc_pthread_create_binary_sem;
 trid_t traceId      = 0;
 trid_t transitionId = 0;
 
+static char resultString[1000] = "***** Model checking completed! *****\n";
```

Comment:
There is perhaps a better C++ method of managing this with `std::string`, though I'm not sure to what extent support looks like with C++11. In any case, this is OK for now I think